activitysim/abm/models/initialize_tours.py

```diff
@@ -94,7 +94,11 @@ def initialize_tours(network_los, households, persons, trace_hh_id):
         model_settings=model_settings.get('annotate_tours'),
         trace_label=tracing.extend_trace_label(trace_label, 'annotate_tours'))
 
-    tours = patch_tour_ids(tours)
+    skip_patch_tour_ids = model_settings.get('skip_patch_tour_ids')
```

Comment:
Is there a standard approach ActivitySim uses for documenting settings that are read from .yaml? Or one that you suggest?

FWIW the `patch_tour_ids()` method itself is not documented and honestly I struggled to discern from the code itself why it was necessary.

Also I did change it just now to make the default evaluate to `False`, that way it will be backwards compatible with existing initialize_tours.yaml files and the user only needs to include this setting if they are aware of it.